Submissions/Ekey_GroceryApp/GroceryApp/GroceryStoreTripModel/FeedbackView.swift

```diff
@@ -0,0 +1,59 @@
+//
+//  FeedbackView.swift
+//  GroceryApp
+//
+//  Created by johnekey on 5/7/19.
+//
+
+import Foundation
+import UIKit
+
+class FeedbackView: UIView {
+    
+    @IBOutlet weak var messageLabel: UILabel!
```

Comment:
What is the difference between adding a comment and starting a review?

Reply:
A review will collect the comments into a single notification, basically.